packages/world/mud.config.mts

```diff
@@ -65,15 +65,15 @@ const config = {
       },
       dataStruct: false,
     },
-    // Bool: {
-    // TODO: This table is only used for testing, move it to `test/tables` via the directory config once supported
-    //   primaryKeys: {},
-    //   schema: {
-    //     value: SchemaType.BOOL,
-    //   },
-    //   storeArgument: true, // TODO Add support for store argument in setter function to table autogen
-    //   tableIdArgument: true,
-    // },
+    Bool: {
+      // TODO: This table is only used for testing, move it to `test/tables` via the directory config once supported
```

Comment:
this should be supported now right?
```suggestion
      directory: "test/tables",
```

Reply:
there're some problems with just doing this, so I left it for another PR, this one is big enough as it is

Reply:
I think forge doesn't see imports in `test`, unless we add `test` to `libs`, which is a questionable decision and I haven't looked into what to really do about it yet

Reply:
I see, fine to leave it for later!

Reply:
(like me, you might wonder how it works in `solecs` - because there `test` is in `src/test`)